src/components/app.tsx

```diff
@@ -234,11 +260,9 @@ export const App = () => {
           </div>
         </div>
       </div>
-      <div className="debug-controls">
-        Playback Rate: { playbackRate >= 1 ? playbackRate : `1/${Math.round(1/playbackRate)}` }x
-        <button onClick={handleFasterPlayback}>Speed up</button>
-        <button onClick={handleSlowerPlayback}>Slow down</button>
-      </div>
+      {/* <div className="current-speed">
+        Speed: { playbackRate >= 1 ? playbackRate : `1/${Math.round(1/playbackRate)}` }x
+      </div> */}
```

Comment:
Personally, I'd rather remove unused code than keep it commented out (unless it's really likely it's gonna come back in the same form and there's a significant amount of work in it).

Reply:
Good catch on the vertical alignment of the sliders; I'll fix that in another commit. 

As for the unused code, I'm in 100% agreement (or normally I would be). But I was under the impression that we would need that code later, so I left it in for now. If we really don't need it, then I'll also delete.